src/test/java/com/ipiecoles/java/java350/service/EmployeServiceIntegrationTest.java

```diff
@@ -59,4 +59,19 @@ public void integrationEmbaucheEmploye() throws EmployeException {
         Assertions.assertEquals(1825.46, employe.getSalaire().doubleValue());
     }
 
+    @Test
+    public void integationCalculPerformanceCommercial() throws EmployeException{
+        //Given
+        Employe employe1 = new Employe("Kara", "Age", "C00001", LocalDate.now(), Entreprise.SALAIRE_BASE, 1, 1.0);
+        employeRepository.save(employe1);
+        Long caTraite = 1500L;
+        Long objectifCa = 1000L;
+
+        //When
+        employeService.calculPerformanceCommercial(employe1.getMatricule(), caTraite, objectifCa);
+
+        //Then
+        Employe employe = employeRepository.findByMatricule(employe1.getMatricule());
+        Assertions.assertEquals(6, employe.getPerformance());
```

Comment:
Il faudrait ici préférer l'utilisation d'AssertJ par rapport à JUnit
`Assertions.assertThat(objectAVerifier).testAEffectuer()`

```suggestion
        Assertions.assertThat(employe.getPerformance()).isEqualTo(6);
```